### PR TITLE
Added hero header component to landing page and updated nav links

### PR DESCRIPTION
**Pull request by viviancan**
Refs #3533 

This change completes the rebranding efforts for the homepage as part of #3159 

I encountered a few issues with using background image in tailwind. After further investigation, I believe it has to do with the wiring between tailwind, webpacker, and the rails asset pipeline. I tried many approaches but had to settle with inline style for the background image. I did not want to add custom styling to any of the old `scss` files since we are proactively moving away from those. I have also attached screenshots for the mobile and desktop view. 

Resources
- [Background image from asset pipeline w/ tailwind](https://stackoverflow.com/questions/66842651/inserting-a-background-image-from-asset-pipeline-with-tailwind)

**Mobile View**
CleanShot 2022-07-14 at 15 32 18@2x

**Desktop View**
CleanShot 2022-07-14 at 15 32 44@2x

### CODE REVIEW

File: app/views/application_rebrand/_global_nav_links.html.erb [hunk 1]
```diff
@@ -15,18 +14,18 @@
 
 <div class="flex items-center justify-center md:hidden">
   <div class="relative inline-block text-left tw-dropdown" id="global-dropdown-wrapper">
-                  <span class="flex cursor-pointer hover:text-energetic-blue"> Menu
-                    <img src="https://icongr.am/fontawesome/caret-down.svg?size=12" alt="Drop down caret">
-                  </span>
+    <span class="flex cursor-pointer hover:text-energetic-blue">
+      <img src="https://icongr.am/fontawesome/bars.svg?size=30" alt="Drop down caret">
```
Comment (viviancan) [inline app/views/application_rebrand/_global_nav_links.html.erb:18]: Updated the dropdown icon to the hamburger menu to match other rebradranding

File: app/views/application/templates/_tg_hero_header.html.erb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<header class="w-full h-96 bg-cover bg-center flex flex-col justify-center"
+        style="background-image: linear-gradient(rgba(0, 0, 0, 0.2), rgba(0, 0, 0, 0.2)), url(<%= image_path "tg-header-landing-page.png"%>);">
```
[the remark this answers is not in the document]
Comment (devkonrad) [reply app/views/application/templates/_tg_hero_header.html.erb:2]: I guess that its a TW2 limitation. I read the tailwind docs about background and they use inline style to set the images too.

**Review by devkonrad — APPROVED** [review]
The changes looks good. Thanks @viviancan !

**Review by shaunxp20 — APPROVED** [review]
LGTM! Some nice changes! Thanks Vivian! 👍 🎉 🙌 💯 :shipit:

**Pull request merged**